core/amber/src/main/python/pytexera/udf/udf_operator.py

```diff
@@ -17,7 +16,7 @@ def open(self) -> None:
 
     @abstractmethod
     def process_tuple(
-        self, tuple_: Union[Tuple, InputExhausted], input_: int
+        self, tuple_: Tuple, input_: int
     ) -> Iterator[Optional[TupleLike]]:
```

Comment:
can you change `input_` to `port` as well? we needed to update it.

Reply:
Fixed.